utils/michigan.py

```diff
@@ -0,0 +1,1052 @@
+        )
+
+        rows_to_fix = rows_to_fix[MICHIGAN_CONTRIBUTION_COLS_REORDER]
+        rows_to_fix.columns = MICHIGAN_CONTRIBUTION_COLS_RENAME
+
+        rows_to_fix["aggregate"] = 0.0
+
+        merged_campaign_dataframe = pd.concat(
+            [merged_campaign_dataframe, rows_to_fix], ignore_index=True
+        )
+
+        return merged_campaign_dataframe
+
+    def drop_menominee_county(
+        self, merged_campaign_dataframe: pd.DataFrame
+    ) -> pd.DataFrame:
+        """Drops the menominee county rows within the Michigan Expenditure data
+
+        Inputs:
+            merged_campaign_dataframe: Pandas DataFrame of merged MI expenditure data
+
+        Returns:
+            merged_campaign_dataframe: Pandas DataFrame of merged MI
+            expenditure data with menominee County Democratic Party columns dropped
+        """
+        # There are only 20 menominee county rows read in incorrectly and
+        # missing key data these rows are dropped
+        subset_condition = (
+            merged_campaign_dataframe["com_type"] == "MENOMINEE COUNTY DEMOCRATIC PARTY"
+        )
+
+        merged_campaign_dataframe = merged_campaign_dataframe.drop(
+            merged_campaign_dataframe.loc[subset_condition].index
+        )
+
+        return merged_campaign_dataframe
+
+    def clean(self, data: list[pd.DataFrame]) -> list[pd.DataFrame]:
+        contribution_dataframe, expenditure_dataframe = data
+
+        clean_cont = self.clean_contribution_dataframe(contribution_dataframe)
+
+        clean_exp = self.clean_expenditure_dataframe(expenditure_dataframe)
+
+        merged_dataframe = pd.concat([clean_cont, clean_exp], axis=0, ignore_index=True)
+        # concatenate the dataframes along rows ignore the prior index
+
+        return [merged_dataframe]
+
+    # NOTE: Helper methods for clean are below
+
+    def clean_contribution_dataframe(
+        self,
+        merged_contribution_dataframe: pd.DataFrame,
+    ) -> pd.DataFrame:
+        """Cleans the contribution dataframe as needed and returns the dataframe
+
+        Inputs:
+                merged_contribution_dataframe:
+                Merged Michigan campaign contribution dataframe
+
+        Returns:
+                merged_contribution_dataframe:
+                Merged Michigan campaign contribution dataframe cleaned in place
+        """
+        merged_contribution_dataframe["cfr_com_id"] = (
+            merged_contribution_dataframe["cfr_com_id"]
+            .apply(pd.to_numeric, errors="coerce")
+            .astype("Int64")
+        )
+        merged_contribution_dataframe["amount"] = merged_contribution_dataframe[
+            "amount"
+        ].apply(pd.to_numeric, errors="coerce")
+        # convert committee IDs to integer, amount and aggregate cols to float
+        merged_contribution_dataframe = merged_contribution_dataframe.drop(
+            columns=[
+                "doc_seq_no",
```

Comment:
you could also put these lists in constants to get rid of ~30 lines between contribution and expenditure functions